src/leopard_em/pydantic_models/managers/refine_template_manager.py

```diff
@@ -109,6 +113,18 @@ def make_backend_core_function_kwargs(
         # The relative pixel size values to search over
         pixel_size_offsets = self.pixel_size_refinement_config.pixel_size_values
 
+        # Load movie and deformation field
+        movie = self.movie_config.movie
+        deformation_field = self.movie_config.deformation_field
+
+        if movie is not None and self.apply_global_filtering:
+            warnings.warn(
+                "Global filtering cannot be applied with movie refinement. "
+                "Disabling apply_global_filtering.",
+                stacklevel=2,
+            )
+            self.apply_global_filtering = False
```

Comment:
It would be preferable to raise a validation error through Pydantic when the movie is not `None` or if `self.movie_config.enabled == True`  and `self.apply_global_filtering ==  True`. That way there are no implicit modifications to a configuration file